languages/anaconda3/template/requirements.yaml

```diff
@@ -0,0 +1,13 @@
+channels:
+# Add your desired anaconda channels here
+- conda-forge
+- defaults
+dependencies:
+# Here are where your anaconda dependencies can be defined
+- python
+# If you wish to use a specific version of pip, please adjust the value below
+- pip=19.0
+- pip:
+# Add your pip dependencies (taken directly from pypi) here, we'll use the version of python and pip installed above
+  - six>=1.12.0
+  - algorithmia>=1.2,<1.3
```

Comment:
Why are we restricting the upper end? I feel like this has bitten us in the past

Reply:
so that we can put potentially breaking changes in 1.3 that won't effect algorithms created in this new handshake - but I agree if we guarenteed that we followed semantic versioning this would be alright